http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/NonLeoMessageSubscriber.scala

```diff
@@ -213,7 +216,16 @@ class NonLeoMessageSubscriber[F[_]](config: NonLeoMessageSubscriberConfig,
   )(implicit ev: Ask[F, AppContext]): F[Unit] =
     for {
       ctx <- ev.ask
-      task = gkeAlg.deleteAndPollNodepool(DeleteNodepoolParams(msg.nodepoolId, msg.googleProject))
+      task = for {
+        _ <- gkeAlg.deleteAndPollNodepool(DeleteNodepoolParams(msg.nodepoolId, msg.googleProject))
+        apps <- appQuery.getAppsByNodepool(msg.nodepoolId).transaction
```

Comment:
Just curious - don't all apps in a nodepool need to be deleted in order for the cronjob to send a `deleteNodepool` message? Just wondering in what circumstance we're deleting a nodepool with non-deleted apps.

Reply:
> Also curious, I was expecting the code to grab the existing apps, then delete the nodepool, then notify the creator that that list of apps had been deleted. (so line 221 comes before line 220)

the code 1. deletes the nodepool in GCP 2. grabs existing apps on the nodepool that's deleted, and tells Sam that they're deleted (that's what `authProvider.notifyResourceDeleted`, 
 there's no notify creator if that's what you're referring to)...